grammars/level8-1-Additions.lark

```diff
@@ -0,0 +1,4 @@
+command: print | ifs elses? | ask | for_loop | assign_list | list_access_var | assign
+
+//new for level 8-1, for loop and the end-blocks
+for_loop: _FOR _SPACE (NAME | var) _SPACE _IS _SPACE (NUMBER | var) _SPACE _TO _SPACE (NUMBER | var) _EOL (_SPACE command) (_EOL _SPACE command)* _EOL "end-block"
```

Comment:
I believe this level was removed somewhat recently.

Reply:
YEs this PR branches of a very old version of main, a new one is in the works